src/Rules.Framework/Generics/GenericRule.cs

```diff
@@ -0,0 +1,40 @@
+namespace Rules.Framework.Generics
+{
+    using System;
+
+    /// <summary>
+    /// Defines a generic rule
+    /// </summary>
+    public class GenericRule
+    {
+        /// <summary>
+        /// Gets the content container which contains the rule content.
+        /// </summary>
+        public object ContentContainer { get; internal set; }
```

Comment:
In this case, you already are exposing the rule content. I think you should rename the property to `Content`, what do you think? Also, review the summary if the name is changed.

Reply:
I agree with renaming, done.